plugins/org.jkiss.dbeaver.ext.clickhouse/src/org/jkiss/dbeaver/ext/clickhouse/model/ClickhouseMetaModel.java

```diff
@@ -63,6 +65,30 @@ public GenericSchema createSchemaImpl(@NotNull GenericDataSource dataSource, @Nu
         return new ClickhouseSchema(dataSource, catalog, schemaName);
     }
 
+    @Override
+    public boolean isSystemSchema(GenericSchema schema) {
+        return schema.getName().equalsIgnoreCase("INFORMATION_SCHEMA") || schema.getName().equals("system");
+    }
+
+    @Override
+    public JDBCStatement prepareTableLoadStatement(
+        @NotNull JDBCSession session,
+        @NotNull GenericStructContainer owner,
+        @Nullable GenericTableBase table,
+        @Nullable String tableName) throws SQLException {
```

Comment:
```suggestion
        @Nullable String tableName
    ) throws SQLException {
```

Reply:
Yes.

Reply:
Nope. I do not love this code style. I love compact code.